test/cypress/support/utils.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const generateRandomId = () => Math.floor(Math.random() * Math.floor(999999999999999));
```

Comment:
This should avoid having two pages with the same id but for sure it's not the best way to do it...